blogposts/2024-05-15-improved-gallery-fabric-0.74.md

```diff
@@ -0,0 +1,103 @@
+
+To get the full list of release details, including breaking changes, check out our [release notes over on our repo.](https://github.com/microsoft/react-native-windows/releases/tag/react-native-windows_v0.74.0)
+
+# ✨ Improved Gallery App
+
+We've been hard at work revitalizing our Gallery app, aligning it with the [WinUI3 gallery](https://www.microsoft.com/store/productId/9P3JFPWWDZRC?ocid=pdpshare) to showcase a more sleek and modern RNW application. In addition, we've ensured it meets inclusive and accessible design requirements and broadened the array of examples to encompass more community modules and common scenarios.
+
+Get React Native Gallery from the [Microsoft store](https://www.microsoft.com/store/productId/9NPG0B292H4R?ocid=pdpshare) or get the latest changes from our (Github repository)[https://github.com/microsoft/react-native-gallery]!
+
+![rn-gallery-gif-2024](assets/gallery-demo-2024.gif)
+
+# 🗺️ Roadmap to Fabric
+
+We've committed ourselves to adding Windows support for React Native's [Fabric](https://reactnative.dev/architecture/fabric-renderer) architecture, the latest rendering system for React Native. Designed for efficiency across platforms in C++, React Native for Windows Fabric seamlessly targets Composition while allowing hosting XAML islands within. Fabric-powered apps default to [WinAppSDK](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/) architecture, aligning with modern [WinUI3 standards](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/). Stay tuned for migration guidance for current UWP RNW apps. Follow our journey to Fabric [here](https://github.com/microsoft/react-native-windows/issues/12042).
+
+*I think it would be super slay to have a demo of an app here. Some choices include Gallery on Fabric (hopefully updated to Chris's latest changes), Playground RNTester, ChatAI App*
+
+## Increasing Core Functionality
+
+We have been hard at work getting component parity up to speed with the old architecture's (*commonly referred to as Paper*) rendering system, ensuring a seamless transition for users migrating to Fabric in the future. Several components, including Views, TextInput, and ActivityIndicator, already support a large set of properties and methods, with more components in the work. Here's a current snapshot of our progress:
+
+| Priority | Component | Available | Properties |
+| --- | --- | --- | --- | 
+| 0 | View| ✅ | 90% | 
+| 0 | Text| ✅ | 85% | 
+| 0 | Image| ✅ | 89% | 
+| 0 | TextInput| ✅ | 80% | 
+| 1 | ScrollView| ✅ | 84% | 
+| 1 | Modal| 🟥 | 64% | 
+| 2 | ActivityIndicator| ✅ | 86% | 
+| 2 | Switch| ✅ | 90% | 
+| 2 | RefreshControl| 🟥 | 28% | 
+*As of February 2024*
```

Comment:
Adding back Chris's feedback to address later "This has been great detail for our more inner circle partners. What's the right detail for the blog audience? I like highlighting that we have components working and most with 80-90% of parity with RNW Paper. But I'm not sure on right way to express that.

We'll want a more up to date snapshot.
I know I need to update the one on the pinned issue, but Jon changed the reporting system so we need to reevaluate."

Reply:
Went though the old issues and updated these to April! I changed RefreshControl and Modal to 0% but it doesn't make a lot of sense as a reader to see that these control aren't implemented but they have properties.